component/azstorage/block_blob.go

```diff
@@ -806,24 +804,24 @@ func (bb *BlockBlob) calculateBlockSize(name string, fileSize int64) (blockSize
 	}
 
 	// If bufferSize <= BlockBlobMaxUploadBlobBytes, then Upload should be used with just 1 I/O request
-	if fileSize <= azblob.BlockBlobMaxUploadBlobBytes {
+	if fileSize <= blockblob.MaxUploadBlobBytes {
 		// Files up to 256MB can be uploaded as a single block
-		blockSize = azblob.BlockBlobMaxUploadBlobBytes
+		blockSize = blockblob.MaxUploadBlobBytes
 	} else {
 		// buffer / max blocks = block size to use all 50,000 blocks
-		blockSize = int64(math.Ceil(float64(fileSize) / azblob.BlockBlobMaxBlocks))
+		blockSize = int64(math.Ceil(float64(fileSize) / blockblob.MaxBlocks))
 
-		if blockSize < azblob.BlobDefaultDownloadBlockSize {
-			// Block size is smaller then 16MB then consider 16MB as default
-			blockSize = azblob.BlobDefaultDownloadBlockSize
+		if blockSize < blob.DefaultDownloadBlockSize {
```

Comment:
This is chaning our check from 16MB to 4MB which means now default block size is smaller.

Reply:
The value of the constant `azblob.BlobDefaultDownloadBlockSize` was always 4MB. The comment was wrong.